app/assets/src/components/ui/containers/ColumnHeaderTooltip.jsx

```diff
@@ -13,10 +13,18 @@ class ColumnHeaderTooltip extends React.Component {
           <div className={cs.tooltip}>
             {title && <span className={cs.title}>{title}:</span>}
             {content}
-            {list &&
-              list.map(item => {
-                return <li key={item}>{item}</li>;
-              })}
+            {list && (
```

Comment:
What does this list contain?

Reply:
This is an unordered list of text for the tooltip to display, for example: 
[
   `The top ${topN} unique taxa per sample, based off relative abundance (rPM)`,
   `Only taxa with at least ${minReads} reads`,
]
![image](https://user-images.githubusercontent.com/53838890/78844274-2895b800-79ba-11ea-9aa1-0bd9ccfec8bc.png)

The component specifies that the proptype should be PropTypes.arrayOf(PropTypes.string)

Reply:
Thanks for clarifying... I was looking to see if there was a more specific name that would make it clearer... but i guess it is a list (that it is a bullet list comes from the code).